lib/app.js

```diff
@@ -28,4 +28,15 @@ exports.getDocuments = getDocuments = function(callback) {
   });
 }
 
+exports.createChannelConfigObject = createChannelConfigObject = function(name, callback) {
+  var channelConfig = require('../resources/channelConfig.json');
+  channelConfig.name = name;
+  channelConfig.urlPattern = "^/CSD/csr/"+ name +"/careServicesRequest$";
```

Comment:
This is better but it won't match if there is anything after careServicesRequest. I think you need to add a `.*` between careServicesRequest and the `$` to make this work.

This site is useful for testing regexes - https://regex101.com/r/xU0iV9/1